x-pack/plugins/security_solution/cypress/e2e/detection_alerts/alerts_charts.cy.ts

```diff
@@ -31,16 +30,16 @@ describe('Histogram legend hover actions', { testIsolation: false }, () => {
 
   it('Filter in/out should add a filter to KQL bar', function () {
     const expectedNumberOfAlerts = 2;
-    cy.get(ALERTS_HISTOGRAM_LEGEND).trigger('mouseover');
-    cy.get(HOVER_ACTIONS.FILTER_FOR).click();
+    cy.get(ALERTS_HISTOGRAM_LEGEND).trigger('mouseover').click();
```

Comment:
Can the `mouseover` just be replaced with `click` for this one and the one on line 41?